client/js/lounge.js

```diff
@@ -645,6 +647,13 @@ $(function() {
 			.sticky()
 			.end();
 
+		chan.find(".line").remove();
```

Comment:
Perhaps instead of removing it, you could find the existing line element and move it in the DOM. This way you're not constantly adding/removing the line marker, and merely just moving it where it needs to be.